massgov/mayflower-patternlab
# Dp 8000 TOC block for exapandable-sections

## Description
This change updates the `table-of-contents-hierarchy` twig to include two blocks and I went ahead and made a new molecule that handles the expand section, `02-molecules/expandable-section.twig`.

## Related Issue / Ticket

- [JIRA issue](https://jira.state.ma.us/browse/DP-8000)


## Steps to Test


1. Go to https://powpow12.github.io/mayflower/?p=templates-binder-page
2. Confirm expandable section in the table of contents still works
3. Go to https://powpow12.github.io/mayflower/?p=organisms-table-of-contents-hierarchy 
4. Confirm expandable section works and when inspecting the patterns you see two new blocks `{% block binderSection %}` and `{% block tocExpandableSection %}`
5. Go to https://powpow12.github.io/mayflower/?p=molecules-expandable-section and confirm the section expands
6. display pattern info and confirm there is a meaningful description.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
